### ISSUE TITLE
Expose MathJax Settings at a course instance level

### ISSUE THREAD

**Issue by Surg-Dev** [PrairieLearn/PrairieLearn#14072]
### Details

For one reason or another, an instructor may dislike PL's visual defaults or want to pre-load some [macros](https://docs.mathjax.org/en/v4.0/input/tex/extensions/configmacros.html#tex-configmacros-options) under tex configuration, or use the [accessibility extension options](https://docs.mathjax.org/en/v4.0/options/accessibility.html) if they know their students need it enabled by default.

Being able to provide a custom MathJax configuration block at a course instance level would be convenient to make these changes without requesting core PL changes that otherwise may conflict with other instructor needs.

The only current work around is to override the MathJax configuration using <script> tags at a per-question basis, and not all configuration options will be updated [_after_ MathJax is loaded.](https://docs.mathjax.org/en/v4.0/input/tex/extensions/configmacros.html#tex-configmacros-options)

The logical extension would be to have a course configuration block, which can be overloaded/shadowed by an instance configuration block. However, the first step would be for course instances as instructors can change from instance to instance, or multiple instructors on one course running different instances.

**Comment by echuber2** [comment]
Further to this, it might make sense if users could customize some of those defaults across their PL account.

**Comment by Surg-Dev** [comment]
> Further to this, it might make sense if users could customize some of those defaults across their PL account.

If such a feature was added, svg/display/[some] accessibility settings can be overridden at any time, which would work per user, but this would need to prevent modifying things like the Tex input processor, since that could just break rendering.

**Comment by rodri76** [comment]
Are you still interested in resolving this? I can work on a fix if it's still available.

**Comment by Surg-Dev** [comment]
I am not actively working on this - be warned though, I don't think this is
a small, self contained change.

**Comment by nwalters512** [comment]
There's not even consensus on _if_ this should be done - @rodri76 remember that the existence of an issue does not imply we're going to do the thing it describes. See "Before you start to code" on https://github.com/PrairieLearn/PrairieLearn/blob/master/CONTRIBUTING.md.
